resources_pages/install_ds_stack_mac.md

```diff
@@ -260,11 +254,11 @@ you can replace `conda install` with `conda install -y`.
 
 ## JupyterLab setup
 
-We will be using the Jupytext Python package and the JupyterLab git extension to facilitate using Jupyter notebooks with Git & GitHub. Install them via the following commands:
+We will be using the Jupytext Python package and the JupyterLab git extension to facilitate using Jupyter notebooks with Git & GitHub. We will also use Jupyter Book to create publication ready documents. Install them via the following commands:
 
 ```bash
-conda install nodejs=15.*
-conda install -c conda-forge jupyterlab jupyterlab-git jupytext
+conda install -c conda-forge jupyterlab-git jupytext jupyter-book jupyterlab-spellchecker
+pip install -U "nbconvert[webpdf]" && pyppeteer-install
```

Comment:
After running this line, I get the following warning message:
```
Installing collected packages: nbconvert
  Attempting uninstall: nbconvert
    Found existing installation: nbconvert 5.6.1
    Uninstalling nbconvert-5.6.1:
      Successfully uninstalled nbconvert-5.6.1
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
myst-nb 0.12.3 requires nbconvert~=5.6, but you have nbconvert 6.1.0 which is incompatible.
Successfully installed nbconvert-6.1.0
```
Also, I see the following in the log file of 
```
MISSING   jupyterlab WebPDF-generation failed. It seems like you have not run `pyppeteer-install` to download chromium for jupyterlab WebPDF export.
```
But when I run `pyppeteer-install` it says
```
[W:pyppeteer.command] chromium is already installed.
```

Reply:
The first part should have no effect, there has been [no significant changes to myst-nb since they added support for higher versions of nbconvert](https://github.com/executablebooks/MyST-NB/commits/master), but it is unfortunate that they didn't update their setup files before releasing because maybe students will think something is wrong now... It should still work to use, could you try running `jupyter-book create mynewbook/` and then `jupyter-book build mybookname/` to confirm that it is working?

For the second part, it is because apple for some reason decided to not include gnu-coreutils as part of bash so the timeout command doesn't exist...  I updated with another solution, can you try again?

I think the nodejs part is fine as either 15 or 16, maybe let's just keep it at 15 for now so that there is no dependency conflicts or some unforeseen edge cases.

Reply:
@armanawn Note that I updated the bash command in my original comment, so try running `bash <(curl -Ss https://raw.githubusercontent.com/UBC-MDS/UBC-MDS.github.io/2021-version-updates/resources_pages/check-setup-mds.sh)` to make sure it uses the latest version of the script when you test

Reply:
I also fixed it so that all error files should be removed now and have their content echoed in the the general log files if there were any erros.

Reply:
@armanawn That error is probably because you nbconvert 5.6.1 instead of of 6.1. Could you check with `jupyter --version` and then try `pip uninstall nbconvert` and `pip install -U "nbconvert[webpdf]"`

Reply:
I added a specific check for nbconvert now so it should also say "missing" in the latest version of the script if it is <6

Reply:
@joelostblom running `pip uninstall nbconvert` and `pip install -U "nbconvert[webpdf]"` solved the problem for me! I did have nbconvert 5.6.1 and now I have 6.1, and no issues anymore. I get all OKs now after running `bash <(curl -Ss https://raw.githubusercontent.com/UBC-MDS/UBC-MDS.github.io/2021-version-updates/resources_pages/check-setup-mds.sh)`.

Reply:
I fixed this by separating the install of pyppeteer from nbconvert in the latest update. No uninstalling should be required now.